fhir-server/src/main/java/com/ibm/fhir/server/util/IssueTypeToHttpStatusMapper.java

```diff
@@ -36,15 +43,20 @@
             for (Issue issue : issues) {
                 IssueType code = issue.getCode();
                 if (code != null && code.getValue() != null) {
-                    return issueTypeToResponseCode(IssueType.ValueSet.from(code.getValue()));
+                    ValueSet issueType = IssueType.ValueSet.from(code.getValue());
```

Comment:
Should this use the new `getValueAsEnumConstant` method?

Reply:
done